cmd/agent/subcommands/run/command.go

```diff
@@ -614,13 +613,7 @@ func startAgent(
 	// start dependent services
 	go startDependentServices()
 
-	if err := otelcollector.Start(); err != nil {
-		return err
-	}
-	// TODO: (components) remove this once migrating the status package to components
-	otlpStatus.SetOtelCollector(otelcollector)
-
-	return nil
+	return otelcollector.Start()
```

Comment:
`otelcollector.Start()` does return an error like the previous version ? or it can't fail anymore ?

Reply:
I'm not entirely sure what you mean. 

Does `otelcollector.Start()` return an error? Yes

> previous version ?

Is there a previous version?

Reply:
Okay my bad all okay, I thinks it is all good for me